src/nvim/options.lua

```diff
@@ -3382,7 +3382,7 @@ return {
         condition = 'MSWIN',
         if_false = 'grep -n $* /dev/null',
         if_true = 'findstr /n $* nul',
-        doc = [["grep -n ",
+        doc = [[Windows: "findstr /n $* nul",
            Unix: "grep -n $* /dev/null"]],
```

Comment:
I don't think that's correct now (since the default can be `ripgrep`). Maybe just "see below" if it's too much to explain here.